tests/ert/unit_tests/config/test_num_cpu.py

```diff
@@ -43,18 +42,23 @@ def test_num_cpu_from_data_file_used_if_config_num_cpu_not_set(parallelsuffix):
     with open(file=data_file, mode="w", encoding="utf-8") as data_file_hander:
         data_file_hander.write(
             f"""
+RUNSPEC
+--comment
+TITLE
+-- A Test Case --
```

Comment:
would suggest not to use commenting symbols in this example line

Reply:
we should possibly parametrize the title to "", "\n", "-- foo", "foo" etc.

Reply:
Added a parametrized list of titles as per our conversation.